lib/bundler/cli/doctor.rb

```diff
@@ -86,9 +88,41 @@ def run
           end
         end.flatten.sort.each {|m| message += m }
         raise ProductionError, message
-      else
+      elsif !permissions_valid
```

Comment:
Question: Is there a `!` too much in this conditional?

Reply:
I personally like how @segiddins implemented the ```permissions_valid``` method to be truthy, and then negated it. I would prefer seeing a truthy method like ```permissions_valid``` with a negation, as opposed to a falsy method (like ```permissions_invalid```), with no negation.

That's just my preference though, and if anyone has a counterargument I'm open to hearing it.